auth/api/http/transport.go

```diff
@@ -46,6 +47,83 @@ func MakeHandler(svc authn.Service, tracer opentracing.Tracer) http.Handler {
 		opts...,
 	))
 
+	mux.Get("/members/:memberID/groups", kithttp.NewServer(
```

Comment:
Please reorder methods in CRUD order (where read = get one, list all, list members, list member of...).